src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -193,35 +260,396 @@ IMF_STD_ATTRIBUTE_DEF (altitude, Altitude, float)
+// camera's white balance settings.
+//
+
+IMF_STD_ATTRIBUTE_DEF (colorTemperature, ColorTemperature, float)
+
+
+//
+// tint - green / magenta tint in effect for the software or
+// hardware creating or capturing the image
+//
+// Positive units of tint correspond to a green shift equal to
+// that effected by a Kodak CC035G or Rosco E-Colour Eighth Plus
+// Green filter; negative units of tint correspond to a magenta
+// shift equal to that effected by a Kodak CC035M or Rosco E-Colour
+// Eighth Minus Green filter.
+//
+// As with colorTemperature, the tint attribute's value is primarily
+// forensic.
+//
+// If the tint attribute is present, the colorTemperature attribute
+// is required to give context to the tint.
+//
+
+IMF_STD_ATTRIBUTE_DEF (tint, Tint, float)
+
+
+//
+// lensMake -- manufacturer or vendor of the lens
+//
+// If present, the value should contain only printable characters
+// and have a nonzero length.
+//
+
+IMF_STD_ATTRIBUTE_DEF (lensMake, LensMake, std::string)
+
+
+//
+// lensModel -- model name or model number of the lens
+//
+// If present, the value should contain only printable characters
+// and have a nonzero length.
+//
+
+IMF_STD_ATTRIBUTE_DEF (lensModel, LensModel, std::string)
+
+
+//
+// lensSerialNumber -- serial number of the lens
+//
+// If present, the value should contain only printable characters
+// and have a nonzero length. Note that despite the name, the
+// string can include non-digits as well as digits.
+//
+
+IMF_STD_ATTRIBUTE_DEF (lensSerialNumber, LensSerialNumber, std::string)
+
+
+//
+// lensFirmwareVersion -- firmware version of the lens
+//
+// If present, the value should contain only printable characters
+// and have a nonzero length.
+//
+
+IMF_STD_ATTRIBUTE_DEF (lensFirmwareVersion, LensFirmwareVersion, std::string)
+
+
+//
+// lensTemperature -- Celsius temperature of the lens at time the
+// image was captured
+//
+
+IMF_STD_ATTRIBUTE_DEF (lensTemperature, LensTemperature, float)
+
+
+//
+// lensAttributes -- lens metadata not specified in other predefined
+// attributes (such as aperture, fNumber, focus, focalLength, lensMake,
+// tStop, etc)
+//
+// This attribute should not be used in place of any predefined
+// lens-related attribute. If present, the string should have a nonzero
+// length.
+//
+
+IMF_STD_ATTRIBUTE_DEF (lensAttributes, LensAttributes, std::string)
+
+
+//
+// focalPlaneImageSize -- width and height in millimeters of the area
+// on the focal plane corresponding to the pixels in the data window,
+// with x defining the width and y defining the height
+//
+
+IMF_STD_ATTRIBUTE_DEF (focalPlaneImageSize, FocalPlaneImageSize, IMATH_NAMESPACE::V2f)
+
+
+//
+// focus -- camera's focus distance, in meters
+//
+// Focus is measured from the camera's film or sensor plane. If the 
+// lens was focused at its hyper-focal setting, the value may be the
+// hyper-focal distance or a positive-infinity floating-point value.
```

Comment:
Indeed the distance is to the focal plane, and as I recall this was because EXIF and other standards did it that way. But EXIF is very vague:

![image](https://user-images.githubusercontent.com/1134845/103463802-36842980-4cfd-11eb-8a22-b0bec2556806.png)

I recall Lars Borg (Adobe) or Jack Hold (HP Labs) saying EXIF was from the focal plane, and that makes sense for consumer photography; plus, the location of the focal plane is often marked on the side of cameras. DPX doesn't carry any sort of subject distance, but the SMPTE Metadata Dictionary (SMPTE RP 210) says "distance from the sensor to the center of interest on the framed subject depicted in the captured essence" as a float. So I don't think the focus semantics were a bad pick. The intent of entrance pupil offset is to allow you to model the lens's center of perspective, and yes, if you are modeling the distance from that point to the subject, you would subtract entrance pupil offset to get the distance from that center of perspective to the subject.

I had thought about moving what seemed to me related attributes together but without any knowledge of how that thing was regarded in our community I held back. Here is my worksheet, with the blueish-gray BG items in the last column of attribute names indicating the ones that got budged. 

[OpenEXR insertion order.xlsx](https://github.com/AcademySoftwareFoundation/openexr/files/5761010/OpenEXR.insertion.order.xlsx)

Reply:
The re-ordering makes sense to me - the proposed organization helps me think about the attributes, as I read the list.